lib/target/ti83p/classic/ti83papp_crt0.asm

```diff
@@ -0,0 +1,294 @@
+;	Stub for the TI 83+ calculator for building as an app
+;
+;  This is preferable to ti83papp.asm
```

Comment:
`ti83papp.asm` just doesn't work anymore - it looks like it was written before we had sections. So feel free to delete or drop this one into its place.